extensions/schema-compare/src/test/schemaCompare.test.ts

```diff
@@ -48,28 +48,31 @@ const mockTargetEndpoint: azdata.SchemaCompareEndpointInfo = {
 	packageFilePath: mocktarget
 };
 
-describe('SchemaCompareDialog.openDialog', function(): void {
-	it('Should be correct when created.', async function(): Promise<void> {
-		let dialog = new SchemaCompareDialog();
-		await dialog.openDialog(mockConnectionProfile);
+describe('SchemaCompareDialog.openDialog', function (): void {
+	it('Should be correct when created.', async function (): Promise<void> {
+		let schemaCompareResult = new SchemaCompareResult();
+		let dialog = new SchemaCompareDialog(schemaCompareResult);
+		await dialog.openDialog();
 
 		should(dialog.dialog.title).equal('Schema Compare');
 		should(dialog.dialog.okButton.label).equal('Ok');
 		should(dialog.dialog.okButton.enabled).equal(false); // Should be false when open
 	});
 });
 
-describe('SchemaCompareResult.start', function(): void {
```

Comment:
thumbs up for updating tests! 
pleaserun intergration ones too - it shouldnt have impact but since this changes the flow...

Reply:
yup integration tests were still passing after this change